src/cli/clear.test.js

```diff
@@ -0,0 +1,134 @@
+// @flow
+
+import path from "path";
+import tmp from "tmp";
+import fs from "fs";
+
+import {makeClear, removeDir, help} from "./clear";
+import {run} from "./testUtil";
+import * as Common from "./common";
+
+describe("cli/clear", () => {
+  describe("'help' command", () => {
+    it("prints usage when given no arguments", async () => {
+      expect(await run(help, [])).toEqual({
+        exitCode: 0,
+        stdout: expect.arrayContaining([
+          expect.stringMatching(/^usage: sourcecred clear/),
+        ]),
+        stderr: [],
+      });
+    });
+
+    it("fails when given arguments", async () => {
+      expect(await run(help, ["foo/bar"])).toEqual({
+        exitCode: 1,
+        stdout: [],
+        stderr: expect.arrayContaining([
+          expect.stringMatching(/^usage: sourcecred clear/),
+        ]),
+      });
+    });
+  });
+
+  describe("'makeClear' command", () => {
+    function throwError() {
```

Comment:
This should actually be returning a promise that rejects, rather than throwing an error, to satisfy the type signature, right?

Reply:
Also, I would move this declaration down to line 100 (just above the tests that use it) to make the code a bit easier to read.

Reply:
Updated (and moved declaration): 

```js
function throwError() {
 return Promise.reject(new Error("test error"));
}
```